emsurfer/.gitignore

```diff
@@ -0,0 +1,44 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
+
+# compiled output
```

Comment:
[yellow]
Mentioned before but one more point here: When you move it, you need to adjust routes. The `vscode` ones you could assume start at the top of the repo I guess.

Reply:
Done